e2e/cypress/tests/integration/accessibility/accessibility_dropdowns_spec.js

```diff
@@ -75,8 +75,6 @@ describe('Verify Accessibility Support in Dropdown Menus', () => {
         menuItems.forEach((item) => {
             // * Verify that the menu item is focused
             if (item === 'View Info' || item === 'Mute Channel' || item === 'Leave Channel') {
-                cy.uiGetChannelMenu().findByText(item).parent().parent().should('be.focused');
```

Comment:
thanks for looking at this, I wasn't sure about what changes may have caused issues here. I think the `if/else` can be removed entirely, so only line 78 is left in the `forEach`. I checked that a moment ago and it appears to assert on each menu item correctly.

Reply:
Oh that was my intent... obviously not what I did tho haha! Thanks

Reply:
Oh that was my intent... obviously not what I did tho haha! Thanks